server/tests/range_test.py

```diff
@@ -18,53 +18,227 @@
+                "Count_Person_65To69Years",
+                "Count_Person_70To74Years",
+                "Count_Person_75To79Years",
+                "Count_Person_80OrMoreYears",
+                "Count_Person_5To17Years",
+                "Count_Person_18To24Years",
+                "Count_Person_25To34Years",
+                "Count_Person_35To44Years",
+                "Count_Person_45To54Years",
+                "Count_Person_55To59Years",
+                "Count_Person_60To61Years",
+                "Count_Person_62To64Years",
+                "Count_Person_65To74Years",
+                "Count_Person_65OrMoreYears",
+                "Count_Person_75OrMoreYears",
+                "Count_Person_5To17Years",
+                "Count_Person_18To24Years",
+                "Count_Person_25To34Years",
+                "Count_Person_35To44Years",
+                "Count_Person_45To54Years",
+                "Count_Person_45To54Years",
+                "Count_Person_55To64Years",
+                "Count_Person_65To74Years",
             ],
-            'Count_Person_65To74Years': ['Count_Person_65To74Years'],
-            'Count_Person_75OrMoreYears': ['Count_Person_75OrMoreYears']
+            "geoId/12345": [
+                "Count_Person_Upto4Years",
+                "Count_Person_5To9Years",
+                "Count_Person_10To14Years",
+                "Count_Person_15To19Years",
+                "Count_Person_20To24Years",
+                "Count_Person_25To29Years",
+                "Count_Person_30To34Years",
+                "Count_Person_35To39Years",
+                "Count_Person_40To44Years",
+                "Count_Person_45To49Years",
+                "Count_Person_50To54Years",
+                "Count_Person_55To59Years",
+                "Count_Person_60To64Years",
+                "Count_Person_65To69Years",
+            ]
+        }
+        expected = {
+            'country/USA': {
+                'Count_Person_Upto9Years': [
+                    'Count_Person_5To9Years', 'Count_Person_Upto4Years'
+                ],
+                'Count_Person_10To19Years': [
+                    'Count_Person_15To19Years', 'Count_Person_10To14Years'
+                ],
+                'Count_Person_20To29Years': [
+                    'Count_Person_25To29Years', 'Count_Person_20To24Years'
+                ],
+                'Count_Person_30To39Years': [
+                    'Count_Person_35To39Years', 'Count_Person_30To34Years'
+                ],
+                'Count_Person_40To49Years': [
+                    'Count_Person_45To49Years', 'Count_Person_40To44Years'
+                ],
+                'Count_Person_50To59Years': [
+                    'Count_Person_55To59Years', 'Count_Person_50To54Years'
+                ],
+                'Count_Person_60To69Years': [
+                    'Count_Person_65To69Years', 'Count_Person_60To64Years'
+                ],
+                'Count_Person_70To79Years': [
```

Comment:
ideally, we'd do 70+ in this case but it's ok for now.

Reply:
Updated to use 70+